### ISSUE TITLE
Graphics for tags and branches are the same : it's misleading

### ISSUE THREAD

**Issue by AlexDLSy** [git-learning-game/oh-my-git#57]
![image](https://user-images.githubusercontent.com/75117170/110663747-05294d80-81c7-11eb-8b27-08c45e853051.png)

It would help the user to differentiate between tags like 'v1' .. 'v3' and 'main' which is a branch, not a tag. 
You can't run "git tag -d main" although main and vx tags show exactly the same in the graph view.

I believe that it would add some readability to introduce a difference in the way they show for UX comfort.

**Comment by abitrolly** [comment]
It would be get the list of all the differences in how `git` treats tags and branches to get the proper mnemonic for them.

**Comment by blinry** [comment]
I agree, @AlexDLSy, thanks! It would really be nice if the icons for tags and branches would be different! :)
